build.py

```
@@ -3,45 +3,146 @@
+        excludes = set()
+
+        # 1. Add all dev packages
+        for dev in self.DEV_PACKAGES:
+            excludes.add(dev)
+
+        # 2. Add bloat if not used
+        for bloat in self.BANNED_BLOAT:
+            if bloat.lower() not in imported:
+                excludes.add(bloat)
+
+        return sorted(list(excludes))
```

Ruff (0.15.12)

[warning] Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)